### PR TITLE
Use new SymPDE's mapped domain

### PR DESCRIPTION
**Pull request by saidctb**
This PR depends on the changes in SymPDE given by PR #71, where we add the notion of a mapped domain and a non-mapped domain is considered as a logical domain.

### CODE REVIEW

**Comment by yguclu** [comment]
It appears that the Travis CI integration is broken for this repository.

**Comment by yguclu** [comment]
I have fixed the Travis CI integration. Does this PR depend on the changes in SymPDE given by PR [#71](https://github.com/pyccel/sympde/pull/71)? That information should be added to this PR's description.

**Comment by saidctb** [comment]
> I have fixed the Travis CI integration. Does this PR depend on the changes in SymPDE given by PR [#71](https://github.com/pyccel/sympde/pull/71)? That information should be added to this PR's description.
yes it depends on them

File: gelato/tests/test_gelatize_3d.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from sympde.core import Constant
 from sympde.calculus import grad, dot, inner, cross, rot, curl, div
 from sympde.calculus import laplace, hessian, bracket, convect
-from sympde.topology import (dx, dy, dz)
+from sympde.topology import dx, dy, dz, dx1, dx2, dx3
```
Comment (yguclu) [inline gelato/tests/test_gelatize_3d.py:13]: `dx, dy, dz` are never used in this module, so they should not be imported.

File: gelato/tests/test_gelatize_2d.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from sympde.core import Constant
 from sympde.calculus import grad, dot, inner, cross, rot, curl, div
 from sympde.calculus import laplace, hessian, bracket, convect
-from sympde.topology import (dx, dy, dz)
+from sympde.topology import dx, dy, dz, dx1, dx2, dx3
```
Comment (yguclu) [inline gelato/tests/test_gelatize_2d.py:13]: `dx, dy, dz` are never used in this module, so they should not be imported.

**Review by yguclu — CHANGES_REQUESTED** [review]
A very good pull request!
I've noticed some unused imports, please remove them...

File: gelato/tests/test_gelatize_3d.py (hunk 1 above)
Comment (yguclu) [reply gelato/tests/test_gelatize_3d.py]: I've done it myself.

File: gelato/tests/test_gelatize_2d.py (hunk 2 above)
Comment (yguclu) [reply gelato/tests/test_gelatize_2d.py]: I've removed them myself.

**Review by yguclu — APPROVED** [review]

**Pull request merged**